bittensor/_neuron/text/template_miner/neuron_impl.py

```diff
@@ -428,6 +433,9 @@ def set_peer_weights( self ):
         except Exception as e:
             logger.error('Failure setting weights on chain with error: {}', e)
 
+    def metagraph_callback(self):
```

Comment:
This is to avoid saving the metagraph as a parameter. Is there a way for us to simply remove it from the parameter set?